build.go

```diff
@@ -1,3 +1,4 @@
+//go:build ignore
```

Comment:
result of `go fmt ./...` -- figured we might as well keep it in to align with the current Go standards